cluster/secrets.go

```diff
@@ -157,3 +157,52 @@ func InstallOrUpdateSecretsByK8SConfig(k8sConfig []byte, orgID uint, query *secr
 
 	return secretSources, nil
 }
+
+// InstallSecretWithVaultID installs a secret which determined by the vaultID to the given namespace
+func InstallSecretWithVaultID(cc CommonCluster, secretID, namespace string) (*secretTypes.K8SSourceMeta, error) {
+	k8sConfig, err := cc.GetK8sConfig()
+	if err != nil {
+		log.Errorf("Error during getting config: %s", err.Error())
```

Comment:
If the error is returned, probably the caller should log it.

Reply:
You can wrap the error if you want:

`fmt.Errorf("error during getting config: %s", err.Error())`

(Also, error messages should be lower cased/should not start with capital letters as per common golang standard)